apps/em/em-npa/prod.yaml

```diff
@@ -8,3 +8,4 @@ spec:
       environment:
         IDAM_API_BASE_URI: "https://idam-api.platform.hmcts.net"
         OPEN_ID_API_BASE_URI: "https://hmcts-access.service.gov.uk/o"
+        S2S_NAMES_WHITELIST: sscs,divorce,ccd,em_gw,xui_webapp
```

Comment:
why do not we change this for env?

Reply:
Details in this ticket https://tools.hmcts.net/jira/browse/EM-5280
It is good practice to first ensure current config is replicated on Prod. Then move to change the config on non-prod environments. Once testing is completed, we will revert the config override from Prod file.

Reply:
you can change the config in test env first?